lib/jekyll-import/importers/blogger.rb

```diff
@@ -267,19 +260,16 @@ def post_data_from_in_entry_elem_info
             { :filename => filename, :header => header, :body => body }
           elsif element_meta[:kind] == "comment"
             timestamp = Time.parse(element_meta[:published]).strftime("%Y-%m-%d")
-            if element_meta[:original_url]
-              @comment_seq ||= 1
+            raise "Original URL is missing" unless element_meta[:original_url]
```

Comment:
if we raise here, will we abort the whole import? would it be better to continue with a warning?

Reply:
OH I see, this is an original `raise`, not something you added. Sorry!